roles/css-injector/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+all_sar_commands: [ 'sar', 'sarl', 'sarm', 'sarv', 'sarjs' ]
```

Comment:
Not going to hold up the PR for this, but the format you're using in roles/change-paths/defaults/main.yml is easier to scan.  So thanks for thinking to make that change above.